src/fileWatcher.ts

```diff
@@ -56,6 +56,7 @@ export namespace ProjectFileWatcher {
                 iProject.setLibraryList(undefined);
 
                 ProjectManager.fire({ type: 'projects', iProject });
+                ProjectManager.fire({ type: 'buildMap', iProject, uri });
```

Comment:
if iproj.json changes does that imply an ibmi.json change?

Reply:
The `buildMap` will store all `version`, `objlib`, and `ccsid` information for each `.ibmi.json` file in the project. In addition, if there is no root `.ibmi.json` or if there is no `objlib` in the root `.ibmi.json`, they will be obtained from the `iproj.json`.

For reference in `updateBuildMap` -> https://github.com/IBM/vscode-ibmi-projectexplorer/blob/main/src/iproject.ts#L347